include/seqan3/range/view/detail.hpp

```diff
@@ -322,6 +317,79 @@ class generic_pipable_view_adaptor : public pipable_adaptor_base<generic_pipable
     {
         return view_type{std::forward<arg_types>(args)...};
     }
+
+public:
+    //!\brief Inherit the base type's constructors.
+    using base_type::base_type;
+};
+
+// ============================================================================
+//  adaptor_from_functor
+// ============================================================================
+
+/*!\brief Template for range adaptor closure objects that store arguments and wrap a proto-adaptor.
+ * \tparam functor_type Type of the proto-adaptor functor.
+ * \tparam stored_args_ts Types of the stored arguments.
+ * \ingroup view
+ *
+ * \details
+ *
+ * This template is particularly useful in combination with range adaptor objects that are not closure
+ * objects (a.k.a. proto adaptors).
+ * These objects take other parameters in addition to the range parameter that they might need store
+ * in which case they can return an object of this type with the stored arguments.
+ *
+ * Additionally, this template delegates back to the proto adaptor for handling range input since
```

Comment:
I don't know what you are trying to say?

Reply:
?

Reply:
I am trying to say: impl() calls operator() of the proto-adaptor since the proto-adaptor has to have an operator() for handling the range anyway.
Is that not clear when looking at the code? :thinking:

Reply:
What formulation would you prefer?

Reply:
Ah ok, this just made it clear. Yes it is obvious from the implementation but I suspected there is more to it when I read the comment. 
 
```suggestion
 * This template delegates back to the proto adaptor for handling range input (impl() calls operator() of the proto-adaptor). 
```
no more. :)